tests/test_codeocean_requests.py

```diff
@@ -646,6 +646,41 @@ def map_to_success_message(input_json: dict) -> dict:
         self.assertEqual(response.content, expected_request_response)
         self.assertEqual(response.status_code, 200)
 
+    @mock.patch("requests.post")
+    def test_update_permissions(
+        self, mock_api_post: unittest.mock.MagicMock
+    ) -> None:
+        """Tests the response of updating permissions"""
+
+        def map_to_success_message(input_json: dict) -> dict:
+            """Map to a success message"""
+            success_message = {}
+            return success_message
+
+        users = [
+                {"email": "user2@email.com", "role": "viewer"}
+            ],
+        groups = [
+                {"group": "group4", "role": "viewer"}
+            ],
+        everyone = "true"
+
+        example_data_asset_id = "648473aa-791e-4372-bd25-205cc587ec56"
+        input_json_data = {"data_asset_id": example_data_asset_id,
+                           "users": users, "groups": groups,
+                           "everyone": everyone}
+
+        mocked_success_post = self.mock_success_response(
+            map_to_success_message, req_type="post"
+        )
+        mock_api_post.return_value = mocked_success_post(json=input_json_data)
+
+        response = self.co_client.update_permissions(
+            data_asset_id=example_data_asset_id, users=users,
+            groups=groups, everyone=everyone
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
Not a huge deal, but the status code is technically a 204

Reply:
Ok set it to 204